server/lib/nl/config_builder/base.py

```diff
@@ -158,9 +205,11 @@ def get_block_strings(self, cv: ChartVars, override_sv: str = ''):
     title, description, footnote = '', '', ''
 
     if override_sv:
-      title = self.sv2thing.name.get(override_sv, '')
-      description = self.sv2thing.description.get(override_sv, '')
-      footnote = self.sv2thing.footnote.get(override_sv, '')
+      title = _make_title_case(self.sv2thing.name.get(override_sv, ''))
+      description = _make_sentence_case(
+          self.sv2thing.description.get(override_sv, ''))
+      footnote = _make_sentence_case(self.sv2thing.footnote.get(
+          override_sv, ''))
```

Comment:
There are various other places we set description and footnote, I wonder if these should apply to the title/description/footnote below too?

I might recommend wrapping the returned (title, description, footnote) into a function that calls ur case-changers and returns them?

Reply:
yes good point. will do in the next PR. removed this functionality for now.